src/Sulu/Component/SmartContent/ContentType.php

```diff
@@ -442,4 +454,19 @@ protected function resolveTags(&$value, $key)
             $value[$key] = $ids;
         }
     }
+
+    private function getUser()
+    {
+        if (!$this->tokenStorage) {
+            return null;
+        }
+
+        $user = $this->tokenStorage->getToken()->getUser();
+
+        if (\is_object($user)) {
```

Comment:
Not sure but think our smart content can only work with UserInterface User so we should only return that kind of users:

```suggestion
        if ($user instanceof UserInterface) {
```